utils/issue-format-bot/.travis.yml

```diff
@@ -0,0 +1,6 @@
+sudo: false
+language: node_js
+node_js:
+  - "7"
+notifications:
+  disabled: true
```

Comment:
For now we can remove this file, since it won't be run within the context of this repo.  I think it should be enough to run the tests manually if the `issue-format-bot` is edited.  I don't think we want to add extra tests to the HTTPS Everywhere repo for utilities.